### PR TITLE
Add option OmitConnResetSession

### PR DESCRIPTION
**Pull request by mniak**
Partial solution to #57

### CODE REVIEW

**Review by XSAM — APPROVED** [review]
Looks good. Please also update the `CHANGELOG.md`.

**Comment by mniak** [comment]
@XSAM changelog updated.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Added
+- Option `OmitResetSession` that if set to true will suppress `sql.conn.reset_session` spans
```
Comment (XSAM) [inline CHANGELOG.md:12]:
```suggestion
### Added

- Option `OmitResetSession` that if set to true will suppress `sql.conn.reset_session` spans. (#87)
```

**Comment by XSAM** [comment]
@mniak Thanks for your contribution.

**Pull request merged**